src/plugins/imagery/components/AnnotationsCanvas.vue

```diff
@@ -164,7 +178,13 @@ export default {
       const targetDetails = [];
       annotations.forEach((annotation) => {
         annotation.targets.forEach((target) => {
-          targetDetails.push(toRaw(target));
+          // only add targetDetails if we haven't added it before
+          const targetAlreadyAdded = targetDetails.some((targetDetail) => {
+            return _.isEqual(targetDetail, toRaw(target));
```

Comment:
```suggestion
            return isEqual(targetDetail, toRaw(target));
```
see above for explicitly importing `lodash/isequal`